test/src/module/command/backupTest.c

```diff
@@ -1661,6 +1661,47 @@ testRun(void)
             "zerofile\n",
             .comment = "copy zero file to repo success");
 
+        // -------------------------------------------------------------------------------------------------------------------------
+        TEST_TITLE("create truncated to zero sized file - checksum will be set but in backupManifestUpdate it will not be copied");
```

Comment:
What is backupManifestUpdate?

Reply:
I copy it from above, I think it means manifestFileUpdate, it was refactored since 1f2ce45e6b613edfb628ac40fd2369c9455692ba